src/app/self-sourced-project/self-sourced-project.component.html

```diff
@@ -1,3 +1,107 @@
-<p>
-  self-sourced-project works!
-</p>
+<h2 class="main-title">Student Placement Arrangement</h2>
```

Comment:
We're using https://material.angular.io/components/categories for our UI components. And we need to make sure that our app looks good on mobile, so it's responsive to screen size. Please have a look at how eoi-business.component.html and eoi-student.component.html are done. Would you be able to do the same? 
To see them in action go to sidenav Student -> Web Site -> Apply. You must be logged in with Google to do so.